renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":label(renovate)", // Add label.
```

Comment:
![must-badge](https://img.shields.io/badge/review-must-red.svg)
The pull request creator is clearly the Renovate bot, so I would like to change it to `dependencies`.
This would make identifying Pull Requests for dependency updates, such as Dependabot security updates, easier.